crates/burn-jit/Cargo.toml

```diff
@@ -15,6 +15,7 @@ default = ["autotune", "std", "burn-compute/default", "fusion"]
 std = []
 doc = ["default"]
 autotune = []
+extension = []
```

Comment:
I would put `template` instead of extension, since you can do a backend extension without it, but it would be with the gpu dialect.